control-plane/csi-controller/src/client.rs

```diff
@@ -31,6 +31,28 @@ pub enum ApiClientError {
     MalformedUrl(String),
 }
 
+/// Placeholder for volume topology for volume creation operation.
+#[derive(Debug)]
+pub struct CreateVolumeTopology {
+    inclusive_label_topology: HashMap<String, String>,
```

Comment:
Would it be better to define a type rather than use `HashMap<String, String>` directly. That way we can change the key/value types in one place if we need to.